### DISCUSSION TITLE
Github Action中拉取语雀文档失败

### DISCUSSION THREAD

**Discussion by SuperYIP** [LetTTGACO/elog#51] [Q&A] [answered]
在本地执行：npm run elog:sync-local可以拉取到语雀文档，但是在Github Action中时提示：缺少语雀配置信息
<img width="1070" alt="action" src="https://github.com/LetTTGACO/elog/assets/32952939/528b1a60-2e82-4a94-8fdb-f897f3b76115">

**Comment by LetTTGACO** [comment, marked as answer]
需要指定环境变量，将本地.elog.env中的变量写入 Github 仓库，文档中有写
https://github.com/elog-x/yuque-vitepress?tab=readme-ov-file#%E9%85%8D%E7%BD%AE%E7%8E%AF%E5%A2%83%E5%8F%98%E9%87%8F
